github-actions/src/main/scala/org/typelevel/sbt/gha/GenerativePlugin.scala

```diff
@@ -210,6 +210,21 @@ object GenerativePlugin extends AutoPlugin {
 ${indent(rendered.mkString("\n"), 1)}"""
     }
 
+  def compileOutputs(outputs: Map[String, String]): String =
```

Comment:
To dogfood this, can we add a couple steps to the sbt-typelevel build that exercise this feature?

Reply:
Ok, I've implemented a couple of jobs that produce and consume `outputs`, the latter failing if the output is missing.